# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1308148424 14400
#      Wed Jun 15 10:33:44 2011 -0400
# Branch stable
# Node ID 07188dbaf89a7847fe9505591d4f0fe946db89d6
# Parent  343eca0298d086765b5a592c90702409c7000aa6
maint: minor update to NEWS file

* NEWS: minor fix in binary incompatibility note.

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -1,34 +1,34 @@
 Summary of important user-visible changes for version 3.4.1:
 -----------------------------------------------------------
 
  ** Octave 3.4.1 is primarily a bug fixing release.
 
  ** IMPORTANT note about binary incompatibility in this release:
 
-    The original goal was to ensure binary compatibility for all 3.4.x
-    releases but this is impossible for the 3.4.1 release due to a bug
-    in the way shared libraries were built in Octave 3.4.0.  Because of
-    this bug, .oct files built for Octave 3.4.0 must be recompiled
-    before they will work with Octave 3.4.1.
+    Binary compatibility for all 3.4.x releases was originally planned,
+    but this is impossible for the 3.4.1 release due to a bug in the way
+    shared libraries were built in Octave 3.4.0.  Because of this bug,
+    .oct files built for Octave 3.4.0 must be recompiled before they
+    will work with Octave 3.4.1.
 
     Given that there would be binary incompatibilities with shared
-    libraries going from Octave 3.4.0 to 3.4.1, we also made the
-    following incompatible changes in this release:
+    libraries going from Octave 3.4.0 to 3.4.1, the following
+    incompatible changes were also made in this release:
 
       * The Perl Compatible Regular Expression (PCRE) library is now
         required to build Octave.
 
       * Octave's libraries and .oct files are now installed in
         subdirectories of $libdir instead of $libexecdir.
 
     Any future Octave 3.4.x release versions should remain binary
-    compatible with Octave 3.4.1 as we are now using proper library
-    versioning as recommended by the libtool manual.
+    compatible with Octave 3.4.1 as proper library versioning is now
+    being used as recommended by the libtool manual.
 
  ** The following functions have been deprecated in Octave 3.4.1 and will
     be removed from Octave 3.8 (or whatever version is the second major
     release after 3.4):
 
       cquad  is_duplicate_entry  perror  strerror
 
  ** The following functions are new in 3.4.1:
